modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -701,3 +702,20 @@ will have the following visual differences compared to the standard `Modal`:
 
 Additionally, users interacting with `Modal` on touch will not be able to exit out of the modal by
 tapping on the overlay.
+
+---
+
+### Popups
+
+We've updated all popup `CloseIcon` and `CloseButton` to default to `type=button`. We did this so
+these close buttons did not submit the forms unintentionally. Without this change, you have to
```

Comment:
```suggestion
these close buttons did not submit when a form element is present. Without this change, you have to
```

Reply:
Feel free to word this any way, but I'm trying to say, when there is a form element in the modal or popup, the close button will not submit, not all instances of popup or modal have forms